specification/azsadmin/resource-manager/InfrastructureInsights/readme.md

```diff
@@ -43,18 +43,9 @@ input-file:
 ---
 # Code Generation
 
-
 ## Swagger to SDK
 
-This section describes what SDK should be generated by the automatic system.
-This is not used by Autorest itself.
-
-``` yaml $(swagger-to-sdk)
-swagger-to-sdk:
-  - repo: azure-sdk-for-go
-```
-
-## C# 
+## C#
 
```

Comment:
There are many code generation settings on this file that I am not sure why they are gone :( They are very useful for us SDK teams

Reply:
We currently will only support C#.  We have no plans for anything else at this time.

Reply:
Why are there no plans for other SDKs yet?

Reply:
Please message my manager.

Reply:
We already talked :D